### DISCUSSION TITLE
Mixtral MoE support?

### DISCUSSION THREAD

**Discussion by stduhpf** [withcatai/node-llama-cpp#126] [Ideas]
As of today, Mixtral models run fine on "vanilla" `llama.cpp`. Is there any plans for supporting this architecture on here too?

**Comment by giladgd** [comment]
@stduhpf I've just released a new version with the latest `llama.cpp`, so it should be supported now

**Reply by giladgd** [reply]
The next beta version will also support it when it comes out

**Reply by stduhpf** [reply]
Works great, thanks!
